src/pymor/analyticalproblems/expressions.py

```diff
@@ -117,6 +117,10 @@ def visit_Constant(self, node):
         return ast.Call(ast.Name('Constant', ast.Load()),
                         [self.generic_visit(node)], [])
 
+    # Python versions prior to 3.8 use ast.Num instead of ast.Constant
```

Comment:
I think this is the first "happens only in a certain python version" issue in a good long while.